DataStructures.Tests/UnitTest1.cs

```diff
@@ -19,16 +19,16 @@ public void LinkedList_starts_empty()
         public void Insert_adds_value_to_start()
         {
             // Arrange
-            LinkedList list = new LinkedList();
+            LinkedListTests list = new LinkedListTests();
 
             // Act
-            list.Insert(1);

```

Comment:
Once you have your using at the top you will be able to access list.Insert(1) properly.